Makefile

```diff
@@ -48,5 +54,9 @@ install:
 
 # Deploy to PyPI
 deploy:
-	python setup.py register
-	twine upload dist/*
+	# python setup.py register
+	twine upload dist/* --verbose
```

Comment:
cross reference this with the btrdb-python repo:

https://github.com/PingThingsIO/btrdb-python/blob/b484d7bc540d951dcd5732708b23ae50996d80a4/.github/workflows/release.yaml#L67-L78

I think sdist and wheels are needed now, but will need to confirm.